README.md

```diff
@@ -0,0 +1,189 @@
+5:Having more secure and performant code.<br>
+
+## Install Linters
+
+You can find linters for most of the programming languages, e.g. Rubocop for Ruby or ESLint for JavaScript.
+
+Also, there are many ways you can integrate a linter in your workflow:
+
+-text editor plugin<br>
+-GitHub Actions<br>
+-GitHub apps<br>
+
+## Set up Linters
+
+**Note:** The npm package manager is going to create a node_modules directory to install all of your dependencies. You shouldn't commit that directory. To avoid that, you can create a .gitignore file and add node_modules to it:
+
+# .gitignore
+
+node_modules/
+
+## Web Hint
+
+This is a customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors.
+
+**NOTE:** If you are using Windows, make sure you initialize npm to create `package.json` file.
+
+```
+npm init -y
+```
+
+1. Run
+   ```
+   npm install --save-dev hint@7.x
+   ```
+   _how to use npm: (https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)._
+2. Copy [.hintrc](.hintrc) to the root directory of your project.
+3. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
+   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
+4. Run
+   ```
+   npx hint .
+   ```
+5. Fix validation errors.
+
+### [Stylelint](https://stylelint.io/)
+
+A mighty, modern linter that helps you avoid errors and enforce conventions in your styles.
+
+1. Run
+
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+not sure how to use npm? Read this.
+
+2. Copy .stylelintrc.json to the root directory of your project.
+
+3. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
+
+If you think that change is necessary - open a Pull Request in this repository and let your code reviewer know about it. 4. Run npx stylelint "\*_/_.{css,scss}" on the root of your directory of your project.
+
+5. Fix linter errors.
+
+6. **IMPORTANT NOTE:** feel free to research auto-correct options for Stylelint if you get a flood of errors but keep in mind that correcting style errors manually will help you to make a habit of writing a clean code!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author**
+
+- GitHub: (https://github.com/migo1)
+- Twitter: (https://twitter.com/@ert49554392)
+- LinkedIn: (https://www.linkedin.com/in/robert-m-3b3b1116a/)
+
+## 🤝 Contributing <a name="contributing"></a>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
```

Comment:
- [ ] I also think it would be better if you improve the content of your `## 🔭 Future Features` with some features to be added later as it is required in all README of your projects. For example:
```
- Add more styling
- Add Animations
- etc
```

Reply:
@tresorsawasawa thanks for the review, please elaborate more on this